website/static/js/pages/view-file-page-dataverse.js

```diff
@@ -0,0 +1,32 @@
+var fileBrowser = require('../fileViewTreebeard.js');
+var nodeApiUrl = window.contextVars.node.urls.api;
+
+$(document).ready(function() {
+    // Treebeard Files view
+    $.ajax({
+        url: nodeApiUrl + 'files/grid/'
+    })
+        .done(function (data) {
+            new fileBrowser(data);
+    });
+});
+
+var panelToggle = $('.panel-toggle');
```

Comment:
Can you add a `TODO` here noting that this should be there is duplication with the Wiki menu here and this should be refactored.